### PR TITLE
Docker Files and Docker-compose

### PR DESCRIPTION
**Pull request by tiago154**

### CODE REVIEW

File: client/README.MD [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+`npm run build`
+
+### Lint
+
+`npm run lint`
+
+##Docker
+
+At the root of the **server folder**:
```
Comment (luanpotter) [inline client/README.MD]: should this be client folder?

**Review by luanpotter — APPROVED** [review]
I think we can un-touch the `client/assets/resources/.keep ` file. it was just to allow me to commit the empty folder. it's no longer empty.

File: client/README.MD (hunk 1 above)
Comment (tiago154) [reply client/README.MD]: famous ctrl c and ctrl v.
I will fix.

**Comment by tiago154** [comment]
> I think we can un-touch the `client/assets/resources/.keep ` file. it was just to allow me to commit the empty folder. it's no longer empty.

I will already correct

**Pull request merged**